native/src/main/java/io/ballerina/stdlib/sql/utils/RecordIteratorUtils.java

```diff
@@ -96,139 +79,6 @@ public static Object nextResult(BObject recordIterator, DefaultResultParameterPr
         }
     }
 
-    private static Object getResult(ResultSet resultSet, int columnIndex, ColumnDefinition columnDefinition,
```

Comment:
Moved this methods to Util

Reply:
Earlier, `nextResult()` was only used within the `RecordIteratorUtils` class. However with `queryRow()`, that method is required when retrieving a single row or value. There is no need to create a create a `RecordIterator` for these two cases. Since it is used in multiple places, thought it might be better to have it in the Utils class.